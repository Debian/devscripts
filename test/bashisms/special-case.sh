#!/bin/sh

case "foo" in
    foo)
	echo once
	;& # BASHISM
    moo)
	echo twice
	;;& # BASHISM
    foo)
	echo foo again
	;;
esac
