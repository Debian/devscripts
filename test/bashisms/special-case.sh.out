possible bashism in bashisms/special-case.sh line 6 (;;& and ;& special case operators):
	;& # BASHISM
possible bashism in bashisms/special-case.sh line 9 (;;& and ;& special case operators):
	;;& # BASHISM
